flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -402,6 +410,20 @@ public void testCassandraTupleAtLeastOnceSink() throws Exception {
 	}
 
 	@Test
+	public void testCassandraRowAtLeastOnceSink() throws Exception {
+		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+		env.setParallelism(1);
+
+		DataStreamSource<org.apache.flink.types.Row> source = env.fromCollection(rowCollection);
+		source.addSink(new CassandraRowSink(INSERT_DATA_QUERY, builder));
```

Comment:
This should use the CassandraRowSinkBuilder to test the row type detection. I know other tests don't do that either, but the should actually.